sklearnex/decomposition/tests/test_pca.py

```diff
@@ -21,6 +21,7 @@
 
 def test_sklearnex_import():
     from sklearnex.decomposition import PCA
-    X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1]])
+    X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
     pca = PCA(n_components=2, svd_solver='full').fit(X)
-    assert 'daal4py' in pca.__module__
+    assert 'sklearnex' in pca.__module__ or 'daal4py' in pca.__module__

```

Comment:
Just a Q: Why the `or`? Shouldn't it be deterministic which version is imported?

Reply:
Tests should be backward compatible